### DISCUSSION TITLE
Value of field variables along a selected path

### DISCUSSION THREAD

**Discussion by abarun22** [idaholab/moose#21494] [Q&A General] [answered]
Dear all,
I am interested in knowing the value of a field variable (possibly for the nodes or discrete points) along a path i selected in the model. I eventually want to plot the response of this variable along this path. Any suggestions here will be appreciated.
Kind regards,
Arun

**Comment by GiudGiud** [comment, marked as answer]
Hello

How is this path? Is it a straight line?
If so you may use this vectorpostprocessor:
https://mooseframework.inl.gov/source/vectorpostprocessors/LineValueSampler.html
If not straight, you will want to create a new one with your custom path off of the LineValueSampler

Guillaume

**Comment by abarun22** [comment]
Hi Guillaume,
Thanks for the reference. It will be a straight path for sure.
This should definitely work.
Kind regards,
Arun
